web/styles/popovers.css

```diff
@@ -568,37 +568,25 @@ ul {
 
     .user-profile-manage-own-edit-button,
     .user-profile-manage-others-edit-button {
-        width: 25px;
-        height: 14px;
-        text-align: center;
-        font-size: 18px;
-    }
-
-    .user-profile-manage-own-copy-link-button {
-        width: 20px;
-        height: 15px;
-        text-align: center;
+        text-decoration: none;
         font-size: 15px;
     }
 
-    #user_profile_manage_own_copy_link {
-        height: 25px;
-        margin-right: 8px;
+    .user-profile-manage-own-copy-link-button {
+        font-size: 19px;
+        text-decoration: none;
     }
 
-    #user_profile_manage_own_edit_link {
-        height: 25px;
+    .zulip-icon-link,
+    .zulip-icon-edit {
+        vertical-align: middle;
     }
 
     .deactivated-user-icon {
         margin: -1px;
         vertical-align: middle;
     }
 
-    .user_profile_name {
```

Comment:
This was improperly not renamed in the previous commit.

I know you're deleting this literally just after, but it's very hard to read your work when there are inconsistencies like this.